### PR TITLE
Add recipe for sqlalchemy_exasol

### PR DESCRIPTION
**Pull request by xhochy**

### CODE REVIEW

**Comment by xhochy** [comment]
@conda-forge/staged-recipes Build is fine, so this is ready for review/merge.

File: recipes/sqlalchemy-exasol/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e55f7de5c383539905ca98d4957297eca8f05a4166ef4024e1634e2e43c9d489
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (scopatz) [inline recipes/sqlalchemy-exasol/meta.yaml]: Can this be `noarch: python`?

**Comment by xhochy** [comment]
@conda-forge/staged-recipes This is ready for review

**Comment by scopatz** [comment]
Thanks @xhochy!

**Pull request merged**